libbeat/_meta/config.reference.yml

```diff
@@ -155,10 +155,10 @@ output.elasticsearch:
   # Number of workers per Elasticsearch host.
   #worker: 1
 
-  # Optional index name. The default is "beatname" plus date
-  # and generates [beatname-]YYYY.MM.DD keys.
+  # Optional index name. The default is "index-prefix" plus date
```

Comment:
should we call it `beat-index-prefix` to make sure `index-prefix` is not used anywhere else and be in sync with the variable name in the Makefile?